theano/tensor/tests/test_opt.py

```diff
@@ -2954,7 +2956,7 @@ def test_local_set_to_inc_subtensor():
 
     # Finally, test that the stack trace is copied over properly,
     # before before and after optimization.
-    assert check_stack_trace(f1, ops_to_check='all')
+    assert check_stack_trace(f1, ops_to_check=tensor.AdvancedIncSubtensor1)
```

Comment:
There is no need to do that on f1, since f1 is compiled without the optimization we want to test, so if it does not work, you can just remove it.
If it works, though, it is slightly better to keep it.